ddl/syncer.go

```diff
@@ -33,6 +36,8 @@ const (
 	// DDLAllSchemaVersions is the path on etcd that is used to store all servers current schema versions.
 	// It's exported for testing.
 	DDLAllSchemaVersions = "/tidb/ddl/all_schema_versions"
+	//DDLServerInformation store DDL server information such as IP,port and so on
```

Comment:
s/store/stores
s/`IP,port`/`IP, port`
Add `.` at the end of a line.

Reply:
And there should be a space between `//` and `DDL...`.